1.raspberry-pi-stuff/service/octopus.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Octopus Recurse application website
+After=network.target
+
+[Service]
+User=root
+Enviroment="FLASK_SECRET_KEY=very very secrete"
+WorkingDirectory=/root
+ExecStart=hypercorn octopus:app --bind 0.0.0.0:5555
```

Comment:
I am not sure what we were speaking about where we want to host the website itself.
but regardless this line is probably missing the internal raspberry pi related services